### ISSUE TITLE
¿how to set a random seed?

### ISSUE THREAD

**Issue by IgnacioRodriguez98** [SciSharp/Keras.NET#248]
Hi there.

I´m came from python and there is necessary to set a random seed to ensure that results are reproducible, how can I set that seed in tensorflow.net?

here are examples to set it in python:

np.random.seed(7)
tf.set_random_seed(7)
